model/config.go

```diff
@@ -1884,6 +1885,10 @@ func (s *TeamSettings) SetDefaults() {
 		s.RestrictCreationToDomains = NewString("")
 	}
 
+	if s.EnableCustomUserStatuses == nil {
+		s.EnableCustomUserStatuses = NewBool(true)
```

Comment:
Shouldn't it be off by default?

Reply:
@reflog I assumed the default value is true by the value set here: https://mattermost.atlassian.net/wiki/spaces/CS/pages/1107034113/Configuration+Setting

But maybe that wasn't meant to be the default value. I think it makes sense to have this off by default.
What do you think?

Reply:
@reflog @chetanyakan the feature should certainly be **enabled by-default**. Admins can disable it if they desire